pkg/aspect/config/quotas.proto

```diff
@@ -32,7 +33,15 @@ message QuotasParams {
         // Map of quota_descriptor label name to attribute expression. At run time each
         // expression will be evaluated to determine the value provided to the aspect. The
         // result of evaluating the expression must match the ValueType of the label in the quota_descriptor.
-        map<string, string> labels = 3;
+        map<string, string> labels = 2;
+
+        // The upper limit for this quota.
+        int64 max_amount = 3;
```

Comment:
I want to leave this as max_amount. Yes, at some point there will be ways to override this value. No matter what this particular field though, it won't convey the fact it can be overridden.

Calling it just "amount" seems confusing to me relative to actual in-use quota amounts.